README.md

```diff
@@ -60,6 +62,34 @@ Kimi-Audio consists of three main components:
 
 ## Quick Start
 
+### 1. Inference with Docker 
+
+We recommend that you build a Docker image to run the inference. You can construct the image using the `docker build` command.
```

Comment:
I believe we can reorganize readme:
- get code ( note you need submodule update --init）
- set envrionment
    - use pre-built image ( we should prefer this because it's simpler)
    - build your own image
- run demo
    - i.e inference with python